webapp/src/serial.tsx

```diff
@@ -422,7 +422,10 @@ class Chart {
 
     addPoint(value: number) {
         this.line.append(Util.now(), value)
-        this.lastUpdatedTime = Util.now()
+        this.lastUpdatedTime = Util.now();
+        // update label with last value
+        const valueText = Number(Math.round(Number(value + "e+2"))  + "e-2").toString();
```

Comment:
What does `Number(Math.round(Number(value + "e+2"))  + "e-2")` do?

Reply:
It's special JS trick to round number to up to 2 decimal numbers.